manager/manager.py

```diff
@@ -48,7 +67,7 @@ def run_command(command: str) -> Tuple[Optional[str], Optional[str]]:
 if latest_tag and latest_tag != data["currentVersion"]:
     # checkout_command = f"git checkout {latest_tag}"
     # run_command(checkout_command)  # checkout the latest tag
-    _, error = run_command("docker-compose -f $HOME/deployment/docker-compose.yml up -d")
+    error = run_command("docker-compose -f $HOME/deployment/docker-compose.yml up -d")
```

Comment:
shouldnt this be changed to docker[space]compose aka podman compose aka compose v2?

Reply:
We still need to use `docker-compose` as we are using the standalone Docker Compose package - https://docs.docker.com/compose/install/standalone/#on-linux